code/src/components/GetThoughts.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import { formatDistance } from 'date-fns';
+
+const GetThoughts = ({ getThoughts, handleLikeButtonOnClick }) => {
+  return (
+    <section className="thoughts wrapper">
+      {getThoughts.map((thought) => (
+        <div
+          className="thoughts card"
+          key={thought._id}>
+          <div className="thoughts-text">
+            <p>{thought.message}</p>
+          </div>
+          <div className="thoughts-likes">
+            <button
+              className="btn-like"
+              type="button"
+              onClick={() => handleLikeButtonOnClick(thought._id)}>
+              <span className="heart">💗</span>
+            </button>
```

Comment:
If you want you also can change the background color of the heart by adding a className and style it in css:

`<button className={(thought.hearts === 0 ? 'heart-button' : 'red-heartButton')} type="button" onClick={() => handleLikeOnClick(thought._id)}>❤️</button>`

```
.heart-button {
	cursor: pointer;
	background-color: #cdbbbb;
	width: 40px;
	height: 40px;
	border-radius: 50px;
	border: 0;
	margin-right: 5px;
}

.red-heartButton {
	cursor: pointer;
	background-color: rgb(191, 105, 105);
	border: 0;
	width: 40px;
	height: 40px;
	border-radius: 50px;
	margin-right: 5px;
}
```